### PR TITLE
React Component for Page Header - RSC-129

### PR DESCRIPTION
**Pull request by rpokorny**
https://issues.sonatype.org/browse/RSC-129

### CODE REVIEW

File: lib/src/components/NxPageHeader/NxPageHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      <span className="nx-page-header__link-sizing-text">{name}</span>
+    </a>
+  );
+}
+
+// visible for testing
+export { HeaderLinkProps, HeaderLink };
+
+export default function NxPageHeader({ links, homeLink, productInfo, children }: Props) {
+  const logoEl = <img src={logoImg} className="nx-product__wordmark" alt="⬡ Sonatype"/>,
```
Comment (glenhunter) [inline lib/src/components/NxPageHeader/NxPageHeader.tsx:34]: Clever, but I don't think the hexagon should be in the alt text

Comment (rpokorny) [reply lib/src/components/NxPageHeader/NxPageHeader.tsx:34]: > Clever, but I don't think the hexagon should be in the alt text

I don't feel too strongly about it, but after double checking the purpose of the alt text I felt like adding this was most faithful to it:

> Content associated with embedded content, to be used when the embedded content cannot itself be used (for example, when the embedded content is of an unsupported format).

**Comment by glenhunter** [comment]
I pushed a branch that shows a method of dealing with the normal/semi-bold sizing issue in the navigation here: https://github.com/sonatype/sonatype-react-shared-components/tree/RSC-129-page-header-component-glen

It doesn't require duplicate `<span>` tags and IMO is much cleaner.

File: lib/src/components/NxPageHeader/NxPageHeader.tsx (hunk 1 above)
Comment (glenhunter) [reply lib/src/components/NxPageHeader/NxPageHeader.tsx:34]: Ok

**Comment by rpokorny** [comment]
> I pushed a branch that shows a method of dealing with the normal/semi-bold sizing issue in the navigation here: https://github.com/sonatype/sonatype-react-shared-components/tree/RSC-129-page-header-component-glen
>
> It doesn't require duplicate <span> tags and IMO is much cleaner.

Accepted.

**Pull request merged**